completions/mfiutil

```diff
@@ -0,0 +1,124 @@
+        fi
+    fi
+
+    local REPLY
+    _comp_count_args -a "-*[Dtu]"
+    case $REPLY in
+        0)
+            # Complete argument to global options
+            case "$prev" in
+                -D)
+                    _comp_compgen_filedir
+                    ;;
+                -t)
+                    case "$OSTYPE" in
+                        *freebsd*|*dragonflybsd*)
+                            _comp_compgen -- -W 'mfi mrsas'
+                            ;;
+                    esac
+                    ;;
+            esac
+            ;;
+        1)
+            _comp_compgen -- -W '
+                version show fail good rebuild syspd drive start abort locate
+                cache name volume clear create delete add remove patrol stop
+                foreign flash bbu ctrlprop'
+            ;;
+        2)
+            case "$prev" in
+                show)
+                    _comp_compgen -- -W '
+                        adapter battery config drives events firmware foreign
+                        logstate volumes patrol progress'
+                    ;;
+                drive)
+                    _comp_compgen -- -W 'progress clear'
+                    ;;
+                start)
+                    _comp_compgen -- -W 'rebuild patrol learn'
+                    ;;
+                abort)
+                    _comp_compgen -- -W 'rebuild'
+                    ;;
+                volume)
+                    _comp_compgen -- -W 'progress'
+                    ;;
+                create)
+                    _comp_compgen -- -W '
+                        jbod raid0 raid1 raid5 raid6 raid10 raid50 raid60 concat'
+                    ;;
+                patrol)
+                    _comp_compgen -- -W 'disable auto manual'
+                    ;;
+                stop)
+                    _comp_compgen -- -W 'patrol'
+                    ;;
+                foreign)
+                    _comp_compgen -- -W 'scan clear diag preview import'
+                    ;;
+                flash)
+                    _comp_compgen_filedir
+                    ;;
+                bbu)
+                    _comp_compgen -- -W 'learn-delay autolearn-mode bbu-mode'
+                    ;;
+                ctrlprop)
+                    _comp_compgen -- -W 'rebuild alarm'
+                    ;;
+            esac
+            ;;
+        3)
+            case "${words[cword-2]}.$prev" in
+                locate.*)
+                    _comp_compgen -- -W 'on off'
+                    ;;
+                cache.*)
+                    _comp_compgen -- -W '
+                        enable disable reads writes write-back write-through
+                        read-ahead bad-bbu-write-cache write-cache'
+                    ;;
+                ctrlprop.alarm)
+                    _comp_compgen -- -W 'on off 1 0'
+                    ;;
+            esac
+            ;;
+    esac
+}
+complete -F _comp_cmd_mfiutil mfiutil mrsasutil
```

Comment:
```suggestion
} &&
    complete -F _comp_cmd_mfiutil mfiutil mrsasutil
```

Reply:
Also, the symbolic link `completions/mrsasutil` should be set up in the make target `symlinks` in `completions/Makefile.am`.